Makefile

```diff
@@ -48,8 +48,8 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr .pytest_cache
 
 lint: ## check style with flake8
-	flake8 src/aioauth tests
-	pyright src/aioauth tests
+	flake8 aioauth tests
```

Comment:
@synchronizing I'm thinking about removing the `flake8` and `black` from the requirements and use only pre-commit.

Reply:
Makes sense to me. I have to revisit the project soon- things took a turn. Whatever is changed in master I'll apply to this branch in time.